github/apps.go

```diff
@@ -36,12 +36,35 @@ type InstallationToken struct {
 	ExpiresAt *time.Time `json:"expires_at,omitempty"`
 }
 
-// InstallationPermissions lists the permissions for metadata, contents, issues and single file for an installation.
+// InstallationPermissions lists the repository and organization permissions for an installation.
+//
+// Permission names taken from:
+//   https://developer.github.com/v3/apps/permissions/
+//   https://developer.github.com/enterprise/v3/apps/permissions/
```

Comment:
This will redirect to the latest Enterprise version.

https://developer.github.com/enterprise/v3/apps/permissions/ -> https://developer.github.com/enterprise/2.17/v3/apps/permissions/